boilerplate/index.ts

```diff
@@ -94,47 +94,47 @@ export function createBoilerplate(sdk: CDP) {
                         terminal.colorRgb(255, 192, 203)('~~~~~ aligned Profile Schema:');
                         terminal('\n');
                         console.log(alignedProfile)
+                        terminal('\n');
 
                     },
 
 
                     async alignActivities() {
+
                         terminal.colorRgb(215, 95, 175)(`~~~~~~~~~ aligning Activities Schema`);
                         terminal('\n');
 
                         let alignActivityPromise: Promise<CustomerSchema>;
-                        const customerSchemas = await bOps.ucpschemas.getAll();
+                        const customerSchemas = await bOps.customerschemas.getAll();
 
                         for (const [activity, boilerplateSchema] of Object.entries(boilerplateActivitySchemas)) {
                             const activitySchema = customerSchemas.find(s => {
                                 return s.name == activity && s.schemaType == SchemaType.Activity
                             });
                             if (!activitySchema) {
-                                alignActivityPromise = bOps.ucpschemas.create({
-                                    enabled: true,
+                                alignActivityPromise = bOps.customerschemas.create({
                                     name: activity,
-                                    schema: JSON.stringify(boilerplateSchema),
+                                    schema: boilerplateSchema,
                                     schemaType: SchemaType.Activity
                                 });
                             } else {
-                                const remoteActivitySchema = JSON.parse(JSON.stringify(activitySchema.schema));
+                                const remoteActivitySchema = activitySchema.schema;
                                 const remoteSchemaProperties = Object.keys(remoteActivitySchema.properties);
                                 const fieldDiffs = Object.keys(boilerplateSchema.properties)
                                     .filter(f => !remoteSchemaProperties.includes(f));
 
 
                                 alignActivityPromise = !fieldDiffs.length ?
                                     Promise.resolve(activitySchema)
-                                    : bOps.ucpschemas.for(activitySchema.id).update({
-                                        enabled: true,
+                                    : bOps.customerschemas.for(activitySchema.id).update({
                                         name: activity,
-                                        schema: JSON.stringify({
-                                            ...remoteActivitySchema,
+                                        schema: {
+                                            ...remoteActivitySchema as Object,
```

Comment:
if the schema now always returns as an object, could you also please change it in the sdk [then you can remove this casting]